qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -226,6 +227,8 @@ class BaseSpecification extends Specification {
     private long testSpecStartTimeMillis
 
     def setupSpec() {
+        MDC.put("logFileName", this.class.getSimpleName())
+        MDC.put("specification", this.class.getSimpleName())
         log.info("Starting testsuite")
```

Comment:
Why do we need it in 2 places?

Reply:
> Why do we need it in 2 places?

I couldn't figure it out but without the extra `.put()` in the `setup()` the keys were cleared. I'll add a comment for that.